src/settings.ts

```diff
@@ -1566,6 +1566,32 @@ export class RemotelySaveSettingTab extends PluginSettingTab {
           });
       });
 
+    new Setting(basicDiv)
+    .setName(t("settings_remoterun"))
```

Comment:
Do you want to open a PR for https://github.com/sboesen/langs with the setting strings for these values (and the dropdown options)? I can take a stab at it if not

I honestly want to merge the two repos but hasn't been a priority.

Reply:
Yep apologies was having trouble with pushing the lang files. I'll open a PR soon.

Reply:
I opened the pull request.